src/openforms/forms/tests/admin/test_views.py

```diff
@@ -0,0 +1,153 @@
+import io
+from unittest.mock import patch
+
+from django.core.files import File
+from django.test import TestCase, override_settings
+from django.urls import reverse
+
+from django_webtest import WebTest
+from privates.test import temp_private_root
+
+from openforms.accounts.tests.factories import SuperUserFactory, UserFactory
+from openforms.forms.models.form import FormsExport
+from openforms.forms.tests.factories import FormFactory
+from openforms.utils.urls import build_absolute_uri
+
+
+@override_settings(LANGUAGE_CODE="en")
+class TestExportFormsView(WebTest):
+    def test_not_staff_cant_access(self):
+        user = UserFactory(is_staff=False, is_superuser=False)
+        self.client.force_login(user)
+
+        response = self.client.get(reverse("admin:forms_export"))
+
+        self.assertEqual(302, response.status_code)
+
+    def test_staff_cant_access(self):
+        user = UserFactory(is_staff=True, is_superuser=False)
+        self.client.force_login(user)
+
+        response = self.client.get(reverse("admin:forms_export"))
+
+        self.assertEqual(403, response.status_code)
```

Comment:
Looks like we also need to update our fixture with the default group permissions to give them access to the export view, no?

Please add a test for a staff user that is not a superuser with the relevant permissions set up. There are some permission-related things already in either the factories or somewhere else in the codebase, you can search for `Permission` model.